### ISSUE TITLE
The "bad practice" example is given first

### ISSUE THREAD

**Issue by BioTurboNick** [MicrosoftDocs/windows-uwp#551]
If Uid/Resw is not the best way, it shouldn't be presented first.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: abca83f8-6801-f90b-9788-041e07967892
* Version Independent ID: 1a3ec3c0-938c-4aa8-e78f-d04dff8168b0
* Content: [Adjust layout and fonts, and support RTL - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/design/globalizing/adjust-layout-and-fonts--and-support-rtl)
* Content Source: [windows-apps-src/design/globalizing/adjust-layout-and-fonts--and-support-rtl.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/globalizing/adjust-layout-and-fonts--and-support-rtl.md)
* Service: **windows**
* GitHub Login: @stevewhims
* Microsoft Alias: **stwhi**

**Comment by stevewhims** [comment]
Thanks, Nicholas, you make a very good point. I've just submitted a correction to the topic (I expect it to be live within the next 24 hours, or end of day Monday at the latest). I've refactored all of the RTL guidance into one section in the topic, and I made sure that we present the alternatives with the most recommended first. As a general principle I wholeheartedly agree that that's the right thing to do. Thanks again! ~Steve (PS I also added a C++/WinRT version of the code example while I was in there).
